### PR TITLE
YouTube Endpoint to Serve Videos

### PR DESCRIPTION
**Pull request by shubhsheth**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [X] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR add this a youtube endpoint to pull videos from CougarCS YouTube channel and show them on the website. 

## Related Tickets & Documents

This PR closes Issue #76 

## Added tests?

- [X] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 docs
- [ ] 📕 storybook
- [X] 🙅 no documentation needed

### CODE REVIEW

File: src/api/routes/youtube.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const router = Router();
+const key = 'youtube';
+
+router.get('/', async (req, res) => {
+	const cacheContent = getCache(key);
+	if (cacheContent) {
+		logger.info('Videos sent from cache');
+		return res.status(200).json(cacheContent);
+	}
```
Comment (vyas0189) [inline src/api/routes/youtube.js:15]: Make sure you write test for these lines: https://coveralls.io/builds/42313127/source?filename=src%2Fapi%2Froutes%2Fyoutube.js#L13

File: test/youtube.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		const res = await agent.get('/api/youtube');
+		expect(res.status).toEqual(200);
+		expect(res.body).toHaveProperty('videos');
+	});
+
+	test('Get videos from cache', async () => {
+		jest.spyOn(apiCall, 'getYoutubeVideos').mockImplementationOnce(
+			() => mockVideos
+		);
+		await agent.get('/api/videos');
```
Comment (vyas0189) [inline test/youtube.test.js:33]: should it be /api/youtube rather than /api/videos

Comment (shubhsheth) [reply test/youtube.test.js]: yes, that's the reason it didn't test the caching lines

**Review by vyas0189 — APPROVED** [review]

**Comment by shubhsheth** [comment]
@KevFlo pinging for review and merge

**Review by KevFlo — APPROVED** [review]

**Pull request merged**